fitwsarah_frontend/src/App.js

```diff
@@ -50,6 +51,7 @@ function App() {
                     <Route path="/trainerFeedback" element={<Feedbacks/>}/>
                     <Route path="/adminFeedback" element={<AdminFeedback/>}/>
                     <Route path="/invoices" element={<ClientInvoices/>}/>
+                    <Route path="/analytics" element={<Analytics/>}/>
```

Comment:
So will the trainer use the same analytics page as the admin?

Reply:
Its the same anyway so why would I just duplicate code